middlewares/addPrefix.go

```diff
@@ -11,7 +12,9 @@ type AddPrefix struct {
 }
 
 func (s *AddPrefix) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	r.URL.Path = s.Prefix + r.URL.Path
+	newPath := s.Prefix + r.URL.Path
+	log.Debugf("Adding Prefix %s to %s = %s", s.Prefix, r.URL.Path, newPath)
```

Comment:
@aantono we think it's not necessary and this can be a problem in some case (If you use some logs tools (like Logstash, Graylog,...) you possibly want to always use debug in production).

Could you remove? 😃